### ISSUE TITLE
cant seem to aggregate based on state

### ISSUE THREAD

**Issue by Russweb** [tfsaggregator/tfsaggregator#155]
I want to aggregate, but not those children with a state(s) of things I dont want.
Like "Removed"

This still aggregates all the children, regardless.

`var mysum =  parent.Children.Where(child => child.Fields["System.State"].Value != "Removed").Sum(child => child.GetField<double>("Microsoft.VSTS.Scheduling.StoryPoints", 0d));`

Bonus: I'd also like to know the syntax if I wanted to do multi filters like 
state != "removed"  AND AnotherField ==  "blah"

**Comment by giuliov** [comment]
I cannot see anything wrong, so I need time to investigate.

**Comment by Russweb** [comment]
would love to know what you find.
It's got me stuck. I simply need to aggregate everything that is not of a certain state.

**Comment by giuliov** [comment]
I have been able to reproduce the issue.
I tried some permutations on the test and the following snippet gives the expected result.

`var mysum = parent.Children.Where(child => "Removed".CompareTo(child.Fields["System.State"].Value) == 0).Sum(child => child.GetField<double>("Microsoft.VSTS.Scheduling.StoryPoints", 0d));`

My gut feeling is that `Value` returns an object and the `==` operator does not map to `string.Equals` but it is too late for deep thinking.
